src/App.js

```diff
@@ -5,26 +5,37 @@ import './App.scss';
 import posts from './api/posts';
 import comments from './api/comments';
 import users from './api/users';
+import { PostList } from './components/PostList/PostList';
+
+const preparedPosts = posts.map(post => ({
+  ...post,
+  user: users.find(user => user.id === post.userId),
+  usersComments: comments.filter(comments => comments.postId === post.id),
```

Comment:
в колбек тебе приходит один комментарий
```suggestion
  comments: comments.filter(comment => comment.postId === post.id),
```